cloud/db_cloud_impl.cc

```diff
@@ -405,6 +393,34 @@ Status DBCloudImpl::DoCheckpointToCloud(
     return st;
   }
 
+  // Copy MANIFEST and CLOUDMANIFEST sequentially only after copying all data
```

Comment:
Is it safe to assume that manifest files will never be updated locally during checkpointing on bulk leaf?

If we can't make that assumption, there might be issue here.

Reply:
This doesn't change what files checkpoint to cloud does. Just the order. If that was a problem, it would be problem without this change too.